x-pack/plugins/index_management/public/plugin.ts

```diff
@@ -31,6 +32,7 @@ export class IndexMgmtUIPlugin {
     plugins: SetupDependencies
   ): IndexManagementPluginSetup {
     const { fleet, usageCollection, management } = plugins;
+    const kibanaVersion = new SemVer(this.ctx.env.packageInfo.version);
```

Comment:
@cjcenizal as you can see, with a system like this in place it is easy to test `7.x` and `8.x` UIs without switching branch (and we know how long that can take).

Replace this line with: `const kibanaVersion = new SemVer('7.16.0')` and that's it.

Reply:
Could we document this process in the readme of the app?

Reply:
I am not sure (yet) that this deserves an entry in the README. It is practical to know this trick to test different UI behaviour on 7.x and 8.x while developing but not sure it is of interest of user testing the app.

Reply:
It might be good to document it maybe in the dev docs. But we probably want to wait and see if the branching system changes in any way with the "make it minor" effort.